src/litserve/__main__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from jsonargparse import set_config_read_mode, set_docstring_parse_options, CLI
```

Comment:
Need to add `jsonargparse[signatures]` to the requirements. Running the cli with just a plain install I get:
```
Traceback (most recent call last):
  File "/home/zeus/miniconda3/envs/cloudspace/bin/litserve", line 5, in <module>
    from litserve.__main__ import main
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/litserve/__main__.py", line 14, in <module>
    from jsonargparse import set_config_read_mode, set_docstring_parse_options, CLI
ModuleNotFoundError: No module named 'jsonargparse'
```

Reply:
Nice catch!